Gemfile

```diff
@@ -118,8 +118,7 @@ gem 'unicorn'
 # Gems we don't depend directly on, but specify here to make sure we don't use a vulnerable
 # version. Please add a link to a security advisory when adding a Gem here.
 
-gem 'i18n', '>=0.6.8'
-# see https://groups.google.com/forum/#!topic/ruby-security-ann/pLrh6DUw998
+gem 'i18n'
```

Comment:
Because people tend to use "bundle update" instead of "bundle (install)", we tend to restrict the versions.